x-pack/plugins/alerting/public/application/maintenance_windows.tsx

```diff
@@ -93,16 +93,12 @@ export const renderApp = ({
         }}
       >
         <Router history={history}>
-          <EuiThemeProvider darkMode={isDarkMode}>
-            <i18nCore.Context>
```

Comment:
The i18n context provider is not explicitly needed since KibnaRenderContextProvider provides it